### ISSUE TITLE
secureOptions: constants.SSL_OP_NO_TLSv1 | constants.SSL_OP_NO_TLSv1_1,

### ISSUE THREAD

**Issue by phanssens1** [blakmatrix/node-zendesk#228]
I'm getting the following issue on version 1.3.0:
```javascript
secureOptions: constants.SSL_OP_NO_TLSv1 | constants.SSL_OP_NO_TLSv1_1,
```

I'm using the following call:
```javascript
client.users.createOrUpdateMany(users, function (err, req, result) {
    if (err) {
      console.log(err);
      return;
    }
    client.jobstatuses.watch(result["job_status"].id, 500, 5, function(err, req, result){
        if (err) {
             console.log(err);
        }
      console.log(JSON.stringify(result, null, 2, true));
    });
  });
```

**Comment by grydstedt** [comment]
Also seeing this

**Comment by erikzrekz** [comment]
@phanssens1 @grydstedt what version of node are you seeing this in?

**Comment by aaronmarkle** [comment]
I am also getting unauthorized errors now that Zendesk has required TLS 1.2

Does anyone know of a quick fix to force the requests to use TLS 1.2?

**Comment by phanssens1** [comment]
I was using 6.10.3 and now i'm using 8.10.0 and it's working fine
